signing/src/main/java/tech/pegasys/web3signer/signing/config/AzureKeyVaultParameters.java

```diff
@@ -25,4 +28,8 @@ public interface AzureKeyVaultParameters {
   String getClientId();
 
   String getClientSecret();
+
+  default Map<String, String> getTags() {
```

Comment:
This default method is only used in one place the AzureSecretSigningMetadata. Would be clearer to just return the emptyMap for that one case.

Reply:
agreed, will make change.